base/codevalidation.jl

```diff
@@ -0,0 +1,149 @@
+# This file is a part of Julia. License is MIT: https://julialang.org/license
+
+const VALID_EXPR_HEADS = Symbol[:call, :invoke, :static_parameter, :line, :gotoifnot, :(=),
+                                :method, :const, :null, :new, :return, :the_exception,
+                                :enter, :leave, :inbounds, :boundscheck, :copyast, :meta]
+
+const ASSIGNED_FLAG = 0x02
+
+struct InvalidCodeError <: Exception
+    errno::Int
+    msg::String
+    meta::Any
+end
+
+InvalidCodeError(errno, msg) = InvalidCodeError(errno, msg, nothing)
+
+"""
+    validate_code!(errors::Vector{>:InvalidCodeError}, c::CodeInfo)
```

Comment:
Because the element type can be "anything", as long as I can push objects of type `InvalidCodeError` into it...at least that was my intent, did I screw something up?